string detectorName
string[] det_attrib
string descriptorName
string[] desc_attrib
sensor_msgs/Image leftImg
sensor_msgs/Image rightImg
---
front_end/frameExtract[] outputFrames
